src/site/markdown/index.md

```diff
@@ -19,102 +19,110 @@
+cache misses. The ultimate target is to find a tradeoff between correctness and performance by configuring cache
+processing rules in an xml file.
 
-In order to achieve better correctness need to:
+To maximize correctness:
 
-* Verify that every relevant file is selected as input to engine
-* Add critical plugin parameters to reconciliation (because they could be overridden from command line)
+* Select every relevant file as input to the engine
+* Add critical plugin parameters to the reconciliation
 
-In order to achieve better reuse need to:
+To maximize reuse need to:
 
-* ensure that non-critical files (test logs, readme and similar) are filtered out from build inputs.
-* non-critical plugin parameters (like number of threads in build)  are filtered out from build inputs
-* Source code is relocatable and build parameters are relocatable (not environment specific)
+* Filter out non-essential files (logs, documentation, and similar)
+* Filter out non-critical plugin parameters (like the number of threads or log level)
+* Make source code relocatable (environment agnostic)
 
-Essentially cache setup is a process of inspecting build, taking these decision and reflect them in the cache
+Effectively, cache setup involves inspecting the build, taking these decisions, and reflecting them in the cache
 configuration.
 
-Please notice though idea of perfectly matching builds might be tempting, but it is not practical with regard to
-caching. Perfect correctness could lead to prevailing hit misses and render caching useless when applied to real
-projects. In practice, configuring sufficient(good enough) correctness might yield the best outcomes. Incremental Maven
-provides flexible and transparent control over caching policy and allows achieving desired outcomes - maximize usability
-or maximize equivalence(correctness) between pre-cached candidates and requested builds.
+Though strict, comprehensive cache rules aiming for 100% coverage of all parameters and files might be tempting, it is
+rarely the optimal decision. When applied to real projects, perfect correctness could lead to prevailing hit misses and
+render caching useless. Configuring sufficient (good enough) correctness might yield the best outcomes. Incremental
+Maven
+provides flexible and transparent control over caching policy and allows achieving desired results - maximizes usability
+or maximize equivalency between pre-cached candidates and requested builds.
 
 ## Usage
 
-Cache extension is an opt-in feature. It is delivered as is and though the tool went through careful verification it's
-still build owner's responsibility to verify build outcomes.
+Cache extension is an opt-in feature. It is delivered as is, and though the tool went through careful verification, it's
+still the build owner's responsibility to verify build outcomes.
 
 ### Recommended Scenarios
 
 Given all the information above, the Incremental Maven is recommended to use in scenarios when productivity and
```

Comment:
`incremental Maven` -> `the build-cache extension`

Reply:
fixed